### ISSUE TITLE
anchor is not defined

### ISSUE THREAD

**Issue by ptica** [danburzo/percollate#19]
```
$percollate pdf --output 1.pdf https://reactjs.org/docs/hello-world.html
Fetching: https://reactjs.org/docs/hello-world.html
Enhancing web page
(node:10750) UnhandledPromiseRejectionWarning: ReferenceError: anchor is not defined
    at Array.from.forEach.img (/usr/lib/node_modules/percollate/src/enhancements.js:12:18)
    at Array.forEach (<anonymous>)
    at imagesAtFullSize (/usr/lib/node_modules/percollate/src/enhancements.js:11:57)
    at cleanup (/usr/lib/node_modules/percollate/index.js:40:2)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:10750) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:10750) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by ptica** [comment]
percollate --version
0.1.5

**Comment by ptica** [comment]
node --version
v10.10.0

**Comment by ptica** [comment]
my global install was a bit bumpy, I had some ancient version of nodejs on my system, installed via sudo, I managed to upgrade using 'n', but puppeteer didn't like the sudo for `npm i -g` however yarn did managed so I sort of only hope my install is ok

**Comment by danburzo** [comment]
Thanks for reporting! Indeed there was a bug with that version, which I had fixed in the meantime. Glad to hear it worked out in the end. I usually install node from with the macOS installer directly, having had trouble with both `brew` and other methods.

I've updated the Readme to reflect what I _think_ is the correct way of upgrading to the latest version using yarn:

```bash
yarn global upgrade --latest percollate
```

I've tested with the example page and it works on percollate@0.2.4, but it seems I still have to write a few more styles for everything on that page to show up correctly in the PDF.

Will get back with updates!
✌️
D.

**Comment by danburzo** [comment]
`percollate@0.2.5` should handle React docs more gracefully. I'm closing this, please feel free to reopen if you still have trouble with the installation and we'll sort it out :-)

**Comment by ptica** [comment]
at ubuntu server of mine I've manually removed the apparently old install of 0.1.5 and did fresh install via yarn and voila: it just works! (after adding some missing libs for pupeteer using https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#chrome-headless-doesnt-launch)

I like the idea very much as pdfs at my reader means much less distractions and is easy to carry around and slowly digest :), thank you!

**Comment by danburzo** [comment]
That's great to hear! I've added a Troubleshooting section to the README that points to the Puppeteer troubleshooting page, thanks for the pointer. 

> I like the idea very much as pdfs on my reader means much less distractions and is easy to carry around and slowly digest :)

That was my thinking as well; I actually got the idea from browsing Simon Wardley's [online book](https://medium.com/wardleymaps), and I wanted to see how the chapters would look when bundled into a single PDF.
